### ISSUE TITLE
Create DITAP Refresh Vendor survey for Workstream 1

### ISSUE THREAD

**Issue by kellysmith1008** [usds/ditap-curriculum-update#323]
As we're working through the content audit to determine next steps for the refresh, we also need to consider vendor feedback and how to incorporate that into the refresh enhancements. 

Tasks:
- [x] Create a survey to send to DITAP delivery agencies, keeping [survey research best practices](https://docs.google.com/document/d/1RwTV2PCrAauDhVNyevUlOgfB2A6MFciiPNhHLQNvjgs/edit?usp=sharing) at the forefront when writing the questions
- [x] [Draft questions for the survey](https://docs.google.com/document/d/1mfiHIYEczkWLmpt5ssv4TD32N2T8gsUeVblsjHmsMKY/edit?tab=t.0)
- [x] Send survey preview to Farooq, Frank, Erin, and Becky
- [x] Make updates and edits to the questions as needed

Acceptance criteria:
- [x] [New survey questions created](https://www.surveymonkey.com/r/Preview/?sm=AtSNR42P2OuKr83WrD_2Fe47_2BhoCyPmR6N90wPlXdNsKBAwJQnAndHRp9bgsjTjTzI)
- [x] Questions shared out with team for feedback
- [x] Questions are edited based on feedback
- [ ] Survey published in SM and ready to collect responses

**Comment by bevasq** [comment]
@kellysmith1008 I think the questions are a good format, thank you for creating this draft. 

@farooqzakhilwal @frank-mcnally re: the last question asking if participants are willing to connect for an interview, do we want to phrase this in a way that encourages them to do this? We would use this as a launching point to start initial/further discussions re: consortium and want to encourage engagement as much as possible. What are your thoughts?

**Comment by farooqzakhilwal** [comment]
@bevasq I agree. I think it would be helpful to frame this almost like it's mandatory to have a follow without saying it's mandatory. Also, for each vendor, it would be important to know who the POC is. How about this instead?

"We value your insights and would love to hear more! We will be scheduling brief 20-minute follow-up interviews with vendors to gather additional insights and foster collaboration on the DITAP Refresh. Please provide the name and email address of the primary contact at your organization for this discussion."

**Comment by kellysmith1008** [comment]
@farooqzakhilwal @bevasq I've updated the last question (wordsmithed a little but the same sentiment) - great suggestion! I will let @Jeff-auser and @frank-mcnally know the survey is ready to send. I'll let them know in slack. Thank you!

**Comment by kellysmith1008** [comment]
[Proposed email to send out the survey](https://docs.google.com/document/d/1dqQVzqZdo_GvIWdM7N7MPdS5pwtTcCMu7z2DJFfZ0p4/edit?usp=sharing):
Subject: We’d love your feedback!
Hi [Name],
We’re working to refresh the DITAP Curriculum, and as a DITAP vendor, your input means a lot to us. Would you mind taking a brief anonymous survey of the DITAP curriculum? It should take between 7-10 minutes to complete and your feedback will play a vital part in our work to enhance the curriculum. Please take a few minutes to complete our quick survey by clicking the link below:
[Take the Survey](https://www.surveymonkey.com/r/86P527F)
Please complete the survey by [MM/DD/YYYY]. No action is needed if you don’t wish to complete our survey. Thank you for your time!
Best regards,
